src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,6 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        return Integer.toBinaryString(value);
```

Comment:
You should write your own implementation

Reply:
Done.